BUILD.bazel

```diff
@@ -0,0 +1,5 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files([
+    "LICENSE",
+])
```

Comment:
What does this do? Why is it needed here?

Reply:
The LICENSE file gets copied from the root into each of the underlying packages and build time. This rule is needed to make the LICENSE file available to subpackage BUILD rules.